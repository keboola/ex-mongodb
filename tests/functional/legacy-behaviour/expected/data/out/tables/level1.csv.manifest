{"incremental":false,"columns":["a","b","level2-a","level2-b","root_pk"],"column_metadata":{"a":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}],"b":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}],"level2-a":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}],"level2-b":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}],"root_pk":[{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"string"}]}}