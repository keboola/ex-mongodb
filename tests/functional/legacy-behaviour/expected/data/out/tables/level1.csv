"123","123","d65f09a7f75f31ad639f3c2b1d7d4d3d","d65f09a7f75f31ad639f3c2b1d7d4d3d","7d0cae8a5690da82b226698d23e94a21"
"123","123","d65f09a7f75f31ad639f3c2b1d7d4d3d","d65f09a7f75f31ad639f3c2b1d7d4d3d","dc6b61323e318be5ec4a277583bdfe0b"
"456","123","77297cc0f4e8f0ea5641dd4b688b331f","77297cc0f4e8f0ea5641dd4b688b331f","81f804cf4fb0e85c4a6563ce5184c92d"
"456","123","77297cc0f4e8f0ea5641dd4b688b331f","77297cc0f4e8f0ea5641dd4b688b331f","0e758a1da8bc9357e64286b1d0fbe2a0"
